src/ubuntu/18.04/amd64/Dockerfile

```diff
@@ -28,9 +28,21 @@ RUN apt-get update \
         npm \
         tar \
         zip \
-    && rm -rf /var/lib/apt/lists/* \
-    && npm install -g azure-cli@0.10.15 \
-    && npm cache clean
+    && rm -rf /var/lib/apt/lists/*
+
+# Install Azure CLI
+RUN apt-get update \
+    && apt-get install ca-certificates curl apt-transport-https lsb-release gnupg -y \
```

Comment:
Consider breaking this apart into multiple lines and alphabetize.  It makes it easier to scan the Dockerfile to see with packages are installed.